cmd/scollector/collectors/processes_linux.go

```diff
@@ -1,18 +1,29 @@
 package collectors
 
 import (
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
```

Comment:
My editor fmts, but apparently not in _linux files. 

I checked build in my linux vm, but guess I should fmt too.